supervisor/conf/config-1.yml

```diff
@@ -0,0 +1,85 @@
+# Connection details for the OpAMP server.
+opampServer:
```

Comment:
Let's use snake_case for config settings. That's what Otel Collector uses.

Reply:
Will do.